.eslintrc

```diff
@@ -21,6 +21,7 @@
         "location"
       ]}],
     "react/display-name" : [0],
+    "react/no-find-dom-node": [0],
```

Comment:
actually the profile page one isn't trivial, not sure what's gong on. if you're cool with it I'd like to leave it off for now.